main.sh

```diff
@@ -67,6 +67,8 @@ echo "::group::Adding files"
 git add -A .
 echo "::endgroup::"
 
+git clean -fxd
```

Comment:
Why did we need a separate clean here? I know there was a reason, but can you remind me?